apisix/plugins/kafka-logger.lua

```diff
@@ -83,6 +83,11 @@ local schema = {
         -- in lua-resty-kafka, cluster_name is defined as number
         -- see https://github.com/doujiang24/lua-resty-kafka#new-1
         cluster_name = {type = "integer", minimum = 1, default = 1},
+        -- config for lua-resty-kafka, default value is same as lua-resty-kafka
+        kafka_batch_num = {type = "integer", minimum = 1, default = 200},
+        kafka_batch_size = {type = "integer", minimum = 0, default = 1048576},
+        kafka_max_buffering = {type = "integer", minimum = 1, default = 50000},
+        kafka_time_linger = {type = "integer", minimum = 1, default = 1}
```

Comment:
Why don't we call it `flush_time`?

Reply:
because this params named “linger.ms” in official kafka client producer and other popular kafka client like sarama,
and i don‘t know why named “flush_time” in resty-kafka......i think call it like “linger” is friendly for user google it....